### DISCUSSION TITLE
Error during call GitHub RestAPI in c# console app (Error : VALUE IS EITHER TOOL LARGE OR TOO SMALL FOR AN INT32)

### DISCUSSION THREAD

**Discussion by v-vermavikas** [community/community#121931] [Programming Help]
### Body

Hello Team,
We are trying to fetch GitHub data using console app using Rest API. But since few days we are getting an issue highlighted as below:

![GetApiCall](https://github.com/community/community/assets/168721012/a4f1dcb5-663d-4790-800b-f41a992a1d62)

We are Making call to api and passing correct repositoryId
![GetAllForRepo](https://github.com/community/community/assets/168721012/481516af-9c66-4836-a4f1-79edd34bc79f)

**Error message: VALUE IS EITHER TOOL LARGE OR TOO SMALL FOR AN INT32****
 Getting an exception value is either too large or too small for an Int32. While passing the repositoryId getting through api and same they are not accepting in another API call. Also, this value is in range.

![thumbnail_image](https://github.com/community/community/assets/168721012/c21ca17b-5684-4c54-9243-b1ba29e72391)

 Could you please help me as mentioned.

Regards,
Vikas

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by kartheekp-ms** [comment]
Upgrading the `octokit` package version to `11.0.1` fixed the issue. https://www.nuget.org/packages/Octokit/11.0.1

Related issue: https://github.com/octokit/octokit.net/issues/2893#issuecomment-2108920988

**Comment by gerdaditeriha** [comment]
We have exactly the same issue now in version 13.0.3, which is latest. has the same problem resurfaced?
